src/plugins/interactive_setup/public/manual_configuration_form.tsx

```diff
@@ -0,0 +1,436 @@
+        </EuiFormRow>
+      )}
+      <EuiSpacer />
+
+      <EuiFlexGroup responsive={false} justifyContent="flexEnd">
+        <EuiFlexItem grow={false}>
+          <EuiButtonEmpty flush="right" iconType="arrowLeft" onClick={onCancel}>
+            <FormattedMessage
+              id="interactiveSetup.manualConfigurationForm.cancelButton"
+              defaultMessage="Back"
+            />
+          </EuiButtonEmpty>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          {state.value ? (
+            <EuiButton
+              type="submit"
+              isLoading={form.isSubmitting}
+              isDisabled={form.isSubmitted && form.isInvalid}
+              color={state.value.statusCode === 200 ? 'warning' : 'primary'}
+              fill
+            >
+              {state.value.statusCode === 200 ? (
+                <FormattedMessage
+                  id="interactiveSetup.manualConfigurationForm.submitWarningButton"
+                  defaultMessage="{isSubmitting, select, true{Connecting to cluster…} other{Connect anyways}}"
+                  values={{ isSubmitting: form.isSubmitting }}
+                />
+              ) : (
+                <FormattedMessage
+                  id="interactiveSetup.manualConfigurationForm.submitButton"
+                  defaultMessage="{isSubmitting, select, true{Connecting to cluster…} other{Connect to cluster}}"
+                  values={{ isSubmitting: form.isSubmitting }}
+                />
+              )}
+            </EuiButton>
+          ) : (
+            <EuiButton
+              type="button"
+              onClick={() => form.setTouched('host')}
+              isLoading={form.isValidating}
+              fill
+            >
+              <FormattedMessage
+                id="interactiveSetup.manualConfigurationForm.validateButton"
+                defaultMessage="{isValidating, select, true{Getting cluster info…} other{Get cluster info}}"
+                values={{ isValidating: form.isValidating }}
+              />
+            </EuiButton>
+          )}
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    </EuiForm>
+  );
+};
+
+export interface CertificatePanelProps {
+  certificate: Certificate;
+}
+
+export const CertificatePanel: FunctionComponent<CertificatePanelProps> = ({ certificate }) => {
+  return (
+    <EuiPanel color="subdued">
+      <EuiFlexGroup responsive={false} alignItems="center" gutterSize="m">
+        <EuiFlexItem grow={false}>
+          <EuiIcon type="document" size="l" />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiTitle size="xxs">
+            <h3>{certificate.subject.O || certificate.subject.CN}</h3>
+          </EuiTitle>
+          <EuiText size="xs">
+            <FormattedMessage
+              id="interactiveSetup.certificatePanel.issuer"
+              defaultMessage="Issued by: {issuer}"
+              values={{
+                issuer: <EuiLink>{certificate.issuer.O || certificate.issuer.CN}</EuiLink>,
```

Comment:
__question:__ why do we need link here?

Reply:
This will show a modal with full certificate details and fingerprint.